examples/src/app/AppRouterAnt.tsx

```diff
@@ -1,26 +1,32 @@
+import { Layout, Row } from 'antd';
 import { inject, observer } from 'mobx-react';
 import * as React from 'react';
+import { NavigationAnt } from '../common/NavigationAnt';
 import { LoginFormAnt } from '../form/LoginFormAnt';
 import { MemTableAnt } from '../memtable/MemTableAnt';
 import { ApplicationAnt } from './ApplicationAnt';
-import { ViewStore } from '../store/ViewStore';
 
-@inject('view')
-@observer
-export class AppRouterAnt extends React.Component<{ view?: ViewStore }> {
-    render() {
-        const view = this.props.view!;
-        return renderContents(view);
-    }
-}
+import { LocationManager } from '@moxb/moxb';
+import { LocationDependentContent } from '@moxb/antd';
 
-function renderContents(view: ViewStore) {
-    switch (view.page) {
-        case 'index':
-            return <ApplicationAnt />;
-        case 'loginForm':
-            return <LoginFormAnt />;
-        case 'memTable':
-            return <MemTableAnt />;
+import { mainMenu, missingContent } from '../MenuStructure.tsx';
+
+@inject('location')
+export class AppRouterAnt extends React.Component<{ location?: LocationManager }> {
+    render() {
+        return (
+            <Layout>
```

Comment:
Here, the idea is that there is a common part of all the pages, and we can render it at a common place (in the code.) Therefore, we have the basic layout here.